x/session/types/params.go

```diff
@@ -2,7 +2,15 @@ package types
 
 import paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 
-var _ paramtypes.ParamSet = (*Params)(nil)
+const (
+	DefaultNumBlocksPerSession = 4
+	NameNumBlocksPerSession    = "num_blocks_per_session"
```

Comment:
What are your thoughts on `ParamNumBlocksPerSession` instead of `NameNumBlocksPerSession`?

This would apply everywhere.

Reply:
I'm down with that.